README.md

```diff
@@ -7,26 +7,26 @@
 
 <img style="text-align:center" src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/smallerlogo.png" alt="drawing" width="120" height="130"/>
 
-The `tfswitch` command line tool lets you switch between different versions of [Terraform](https://www.terraform.io/).  
-If you do not have a particular version of Terraform installed, `tfswitch` will download and verify the version you desire.  
-The installation is minimal and easy.  
+The `tfswitch` command line tool lets you switch between different versions of [Terraform](https://www.terraform.io/).
+If you do not have a particular version of Terraform installed, `tfswitch` will download and verify the version you desire.
+The installation is minimal and easy.
 Once installed, simply select the version you require from the dropdown and start using Terraform.
 
 ## Documentation
 Click [here](https://tfswitch.warrensbox.com) for our extended documentation.
 
 ## NOTE
-Going forward we will change the version identifier of `tfswitch` to align with the common go package versioning.  
-Please be advised to change any automated implementation you might have that is relying on the `tfswitch` version string.  
-**Old version string:** `0.1.2412`  
+Going forward we will change the version identifier of `tfswitch` to align with the common go package versioning.
+Please be advised to change any automated implementation you might have that is relying on the `tfswitch` version string.
+**Old version string:** `0.1.2412`
```

Comment:
Please put two trailing periods back — they are line breaks in markdown.

Reply:
Fixed